Controller/Adminhtml/Support/ConfigurationSettingsForm.php

```diff
@@ -1,14 +1,27 @@
 <?php declare(strict_types=1);
 
 namespace Adyen\Payment\Controller\Adminhtml\Support;
+
 use Magento\Backend\App\Action;
 use Magento\Backend\App\Action\Context;
 use Magento\Framework\Controller\ResultFactory;
+use Adyen\Payment\Helper\SupportFormHelper;
 
 class ConfigurationSettingsForm extends Action
 {
-    public function __construct(Context $context)
+    const CONFIGURATION_SETTINGS_EMAIL_TEMPLATE = 'configuration_settings_email_template';
+
+    /**
+     * @var SupportFormHelper
+     */
+    protected SupportFormHelper $supportFormHelper;
```

Comment:
```suggestion
    protected $supportFormHelper;
```
Type declaration should be removed if we are to offer this functionality on v8, since we are still supporting php 7.3 on it